packages/platform-android/src/commands/runAndroid/runOnEmulator.ts

```diff
@@ -26,29 +26,62 @@ function toPascalCase(value: string) {
 
 type AndroidProject = NonNullable<Config['project']['android']>;
 
-async function runOnAllDevices(
+function logStartEmulatorError(error: Error) {
```

Comment:
Idea: How about we actually throw `CLIError` in all cases here, instead of custom logging / error handling logic?